packages/components/src/components/icon/library.system.ts

```diff
@@ -22,7 +22,15 @@ export const icons = {
   <svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
     <path d="M21.707 2.293a1 1 0 0 0-1.414 0L12 10.586 3.707 2.293a1 1 0 0 0-1.413 1.414l-.001-.001 8.293 8.293-8.293 8.293a1 1 0 0 0 0 1.414 1 1 0 0 0 1.414 0L12 13.413l8.293 8.293a1 1 0 0 0 1.414 0 1 1 0 0 0 0-1.414l-8.293-8.293 8.293-8.293a1 1 0 0 0 0-1.414z"/></svg>
   </svg>
-`
+`,
+  start: `
+  <svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="m19.54 11.16-14-9c-.153-.099-.339-.158-.54-.158-.552 0-.999.447-1 .998v18c0 .552.448 1 1 1 .201-.001.387-.06.544-.162l-.004.002 14-9c.271-.182.447-.487.447-.833s-.176-.652-.444-.831l-.004-.002zm-13.54 8.007v-14.333l11.15 7.167z"/></svg>
+  `,
+  pause: `
+    <svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path d="M9 22a.99997.99997 0 0 0 1-1V3a1.00003 1.00003 0 0 0-1-1H5a1.00003 1.00003 0 0 0-1 1v18a.99997.99997 0 0 0 1 1h4ZM6 4h2v16H6V4ZM19 22c.2652 0 .5196-.1054.7071-.2929A1.0001 1.0001 0 0 0 20 21V3a.99997.99997 0 0 0-1-1h-4a.99997.99997 0 0 0-1 1v18c0 .2652.1054.5196.2929.7071S14.7348 22 15 22h4ZM16 4h2v16h-2V4Z"/>
+      </svg>
```

Comment:
These are the correct new icon names, right?

Reply:
Yes!